tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
```

Comment:
Yeah, I *tried* that. However, it didn't succeed. I tried logging to stdout/stderr and check the output after the call, but sometimes the log-lines aren't in the output. That's the cause of this rather ugly thing here.